### PR TITLE
Grey out fill button when resource maxed out

### LINKED ISSUE #32: grey out fill/repair buttons in hangar node if not needed
- Waiting on UI Mockups

### PR DESCRIPTION
**Pull request by maxwellpark**
Grey is the new black.

### CODE REVIEW

File: Assets/Scripts/UI/Hangar/HangarNodeUI.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public enum HangarPanel
 
     public Slider fuelSlider;
     public FuelButton fuelButton;
+    private Button fuelButtonControl;
```
Comment (ambid17) [inline Assets/Scripts/UI/Hangar/HangarNodeUI.cs]: just access the button from FuelButton, no need to grab another variable

File: Assets/Scripts/UI/Hangar/HangarNodeUI.cs [hunk 2]
```diff
@@ -113,6 +120,11 @@ private void CheckFueling()
             shipToInspect.currentFuel++;
             fuelSlider.value = shipToInspect.GetFuelPercent();
             fuelTimer = 0;
+            
+            if (shipToInspect.currentFuel >= shipToInspect.maxFuel)
```
Comment (ambid17) [inline Assets/Scripts/UI/Hangar/HangarNodeUI.cs]: should probably do this if we dont have the money too

**Review by ambid17 — CHANGES_REQUESTED** [review]

File: Assets/Scripts/UI/Hangar/HangarNodeUI.cs (hunk 1 above)
Comment (maxwellpark) [reply Assets/Scripts/UI/Hangar/HangarNodeUI.cs]: ok, we do have to call getcomponent multiple times this way though

Comment (ambid17) [reply Assets/Scripts/UI/Hangar/HangarNodeUI.cs]: make it grab a reference to itself on awake. one getcomponent call. access it with fuelButton.button

**Review by ambid17 — APPROVED** [review]

**Pull request merged**